File: JavaScript/1791_Find_Center_of_Star_Graph/solution.js
/**
 * @param {number[][]} edges
 * @return {number}
 */
const findCenter = function (edges) {
    const firstEdge = edges[0];
    const secondEdge = edges[1];

    if (secondEdge.includes(firstEdge[0])) {
        return firstEdge[0];
    }
    else {
        return firstEdge[1];
    }
};

File: Go/1791_Find_Center_of_Star_Graph/solution.go
package solution

func findCenter(edges [][]int) int {
	firstEdge := edges[0]
	secondEdge := edges[1]

	if firstEdge[0] == secondEdge[0] || firstEdge[0] == secondEdge[1] {
		return firstEdge[0]
	} else {
		return firstEdge[1]
	}
}
